homeassistant/components/vizio/media_player.py

```diff
@@ -135,7 +163,7 @@ def __init__(
     @util.Throttle(MIN_TIME_BETWEEN_SCANS, MIN_TIME_BETWEEN_FORCED_SCANS)
     async def async_update(self) -> None:
         """Retrieve latest state of the device."""
-        is_on = await self._device.get_power_state(False)
+        is_on = await self._device.get_power_state(self._timeout, False)
```

Comment:
I would expect to be able to set a default timeout when instantiating the vizio library device. Having to pass the timeout on every method call seems too verbose.